Library/Homebrew/cmd/uses.rb

```diff
@@ -67,7 +67,10 @@ def uses
       opoo e
       used_formulae_missing = true
       # If the formula doesn't exist: fake the needed formula object name.
+      # This is a legacy use of OpenStruct that should be refactored.
```

Comment:
Ideally: maybe suggest what would be a better fit than `OpenStruct` here (and elsewhere)?